Sena/doc/create sensor description (#449)
autowarefoundation/autoware-documentation: docs/how-to-guides/integrating-autoware/creating-vehicle-and-sensor-description/creating-vehicle-description/creating-vehicle-description.md

This parameter file contains the dimensions of the mirrors on the sides of the vehicle.
Used in the [crop-box-filter](https://autowarefoundation.github.io/autoware.universe/main/sensing/pointcloud_preprocessor/docs/crop-box-filter/) of
[PointCloudPreprocessor](https://autowarefoundation.github.io/autoware-documentation/main/design/autoware-architecture/sensing/data-types/point-cloud/).

The original file is [mirror.param.yaml](https://github.com/autowarefoundation/sample_vehicle_launch/blob/main/sample_vehicle_description/config/mirror.param.yaml)
included in [sample_vehicle_launch](https://github.com/autowarefoundation/sample_vehicle_launch/tree/main) and given below.

??? note "mirror.param.yaml for sample_vehicle_launch"

    ```yaml
    /**:
      ros__parameters:
        min_longitudinal_offset: 1.8
        max_longitudinal_offset: 3.2
        min_lateral_offset: -1.4
        max_lateral_offset: 1.4
        min_height_offset: 0.8
        max_height_offset: 1.5
    ```

!!! Warning

    Since the tutorial vehicle does not have a mirror, the parameters are set to 0.0.

### 2. simulator_model.param.yaml

Configuration file for the simulator environment.

> The original file is [simulator_model.param.yaml](https://github.com/autowarefoundation/sample_vehicle_launch/blob/main/sample_vehicle_description/config/simulator_model.param.yaml)
> included in [sample_vehicle_launch](https://github.com/autowarefoundation/sample_vehicle_launch/tree/main) and given below.

Reviewer: ```suggestion
The default [simulator_model.param.yaml](https://github.com/autowarefoundation/sample_vehicle_launch/blob/main/sample_vehicle_description/config/simulator_model.param.yaml) file for sample_vehicle_launch is given below.
```

    ```yaml
    <?xml version="1.0"?>
    <robot xmlns:xacro="http://ros.org/wiki/xacro">
      <!-- load parameter -->
      <xacro:property name="vehicle_info" value="${xacro.load_yaml('$(find sample_vehicle_description)/config/vehicle_info.param.yaml')}"/>

      <!-- vehicle body -->
      <link name="base_link">
        <visual>
          <origin xyz="${vehicle_info['/**']['ros__parameters']['wheel_base']/2.0} 0 0" rpy="${pi/2.0} 0 ${pi}"/>
          <geometry>
            <mesh filename="package://sample_vehicle_description/mesh/lexus.dae" scale="1 1 1"/>
          </geometry>
        </visual>
      </link>
    </robot>
    ```

## 2. VEHICLE-ID_vehicle_launch

`vehicle_interface.launch.xml` is the launch file related to the drive system. Please modify it according
to the configuration of your vehicle's drive system.

### Folder Structure

??? note "VEHICLE-ID_vehicle_description Folder Structure"

    ```diff
    VEHICLE-ID_vehicle_launch/
       └─ launch/
             └─ vehicle_interface.launch.xml
    ```

If you are operating multiple vehicles, use the vehicle_id to switch to the corresponding configuration for each vehicle.

> The original file is [vehicle_interface.launch.xml](https://github.com/autowarefoundation/sample_vehicle_launch/blob/main/sample_vehicle_launch/launch/vehicle_interface.launch.xml)
> included in [sample_vehicle_launch](https://github.com/autowarefoundation/sample_vehicle_launch/tree/main) and given below.

Reviewer: ```suggestion
The default file is [vehicle_interface.launch.xml](https://github.com/autowarefoundation/sample_vehicle_launch/blob/main/sample_vehicle_launch/launch/vehicle_interface.launch.xml) for sample_vehicle_launch is given below.
```

??? note "vehicle_interface.launch.xml for sample_vehicle_launch"

    ```yaml
    <?xml version="1.0" encoding="UTF-8"?>
    <launch>
      <arg name="vehicle_id" default="$(env VEHICLE_ID default)"/>
    </launch>1

>     ```

Reviewer: So small file, so I think there is no need `note` as well.
